ecl/hql/hqlexpr.cpp

```diff
@@ -16188,6 +16618,292 @@ IHqlExpression * annotateIndexBlobs(IHqlExpression * expr)
     return transformer.transform(expr);
 }
 
+/*The following confusion  matrix holds the cost of transpositions and substitutions for weighted-Damerau-Levenshtein Distance.
```

Comment:
This should be deleted, and this PR should be based on the branch that contains the code adding it to stringlib.